server/plugin.go

```diff
@@ -172,15 +171,22 @@ func (p *Plugin) GetURL() string {
 	return siteURL + "plugins/" + pluginID
 }
 
-func (p *Plugin) GetRelativeURL() string {
-	config := p.API.GetConfig()
+func (p *Plugin) GetURL() string {
+	return getURL(p.API.GetConfig())
+}
```

Comment:
Testing these without a mock proved tricky because of how the server intervenes and changes the SiteURL. So I opted to refactor and just test `getURL` and `getRelativeURL` directly from passed `*model.Config` parameters.